simtools/applications/print_array_elements.py

```diff
@@ -145,6 +145,13 @@ def _parse(label=None, description=None):
         default=None,
         nargs="+",
     )
+    config.parser.add_argument(
```

Comment:
Not sure it justifies adding a parameter to skip the validation, which is necessary only in the very specific case of longitude/latitude? Isn't it better to implement an if statement in the code and exclude the validation when longitude/latitude is used?

Reply:
I do think this is useful for other cases. We use this application a lot with input from quickly generated files. In many cases, the metadata is not complete (which is fine, as these are temporary fine) and maybe not all columns are there. So this is useful to have for this application.

Reply:
ok. this should not become default though.